_updatePublisher_curl.sh

```diff
@@ -32,7 +32,7 @@ done
 
 echo "Checking internet connection"
 case "$OSTYPE" in
-  linux-gnu* ) curl -sSf tx.fhir.org > /dev/null ;;
+  linux* ) curl -sSf tx.fhir.org > /dev/null ;;
```

Comment:
do we still need both updatePublisher and updatePublisher_curl ?

Reply:
I still have the issue with ping :/